instance/tests/models/test_utils.py

```diff
@@ -486,3 +486,35 @@ class DjangoResourceTest(SimpleResourceTestCase):
     Run the same tests as in SimpleResourceTestCase, but using DjangoResource.
     """
     make_resource = DjangoResource
+
+    def setUp(self):
+        self.make_resource.save = Mock()
```

Comment:
@bradenmacdonald Hm, using `patch.object` doesn't work because `DjangoResource` doesn't define a `save` method itself. Now that `ModelResourceStateDescriptor._set_state` is responsible for persisting the current status of a resource, most of the tests that `DjangoResourceTest` inherits from `SimpleResourceTestCase` depend on the `save` method being available, but they are agnostic to what it actually does. So we could add a no-op `save` method to `DjangoResource` and patch it in `DjangoResourceTest` as you suggest, but this would be kind of redundant. Alternatively we could keep the existing implementation -- that shouldn't do any harm since `DjangoResource` is only being used by `DjangoResourceTest`.